### PR TITLE
chore(deps): bump

### PR DESCRIPTION
**Pull request by kanadgupta**
## 🧰 Changes

Before:

```
Package                Current  Wanted  Latest  Location                            Depended by
@readme/eslint-config   10.3.2  10.4.5  10.4.5  node_modules/@readme/eslint-config  rdme
@types/jest             29.2.5  29.2.6  29.2.6  node_modules/@types/jest            rdme
chalk                    4.1.2   4.1.2   5.2.0  node_modules/chalk                  rdme
config                   3.3.8   3.3.9   3.3.9  node_modules/config                 rdme
configstore              5.0.1   5.0.1   6.0.0  node_modules/configstore            rdme
eslint                  8.31.0  8.32.0  8.32.0  node_modules/eslint                 rdme
jest-extended            3.2.1   3.2.3   3.2.3  node_modules/jest-extended          rdme
nock                    13.2.9  13.3.0  13.3.0  node_modules/nock                   rdme
node-fetch               2.6.7   2.6.8   3.3.0  node_modules/node-fetch             rdme
oas                     20.3.0  20.4.0  20.4.0  node_modules/oas                    rdme
ora                      5.4.1   5.4.1   6.1.2  node_modules/ora                    rdme
prettier                 2.8.1   2.8.3   2.8.3  node_modules/prettier               rdme
simple-git              3.15.1  3.16.0  3.16.0  node_modules/simple-git             rdme
ts-jest                 29.0.3  29.0.5  29.0.5  node_modules/ts-jest                rdme
type-fest                3.5.1   3.5.2   3.5.2  node_modules/type-fest              rdme
```

After:
```
Package      Current  Wanted  Latest  Location                  Depended by
chalk          4.1.2   4.1.2   5.2.0  node_modules/chalk        rdme
configstore    5.0.1   5.0.1   6.0.0  node_modules/configstore  rdme
node-fetch     2.6.8   2.6.8   3.3.0  node_modules/node-fetch   rdme
ora            5.4.1   5.4.1   6.1.2  node_modules/ora          rdme
```

### CODE REVIEW

File: src/cmds/validate.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import type { CommandOptions } from '../lib/baseCommand';
 import type { Options } from './openapi/validate';
+import type { CommandOptions } from '../lib/baseCommand';
```
Comment (erunion) [inline src/cmds/validate.ts:2]: weird, this reorder shouldn't have happened.

Comment (kanadgupta) [reply src/cmds/validate.ts:2]: lol yikes... could this be another bug with that import plugin? i can hold off

Comment (erunion) [reply src/cmds/validate.ts:2]: Looks like it's just `type` imports that are this way. I'll update that ticket I had

Comment (erunion) [reply src/cmds/validate.ts:2]: Looks like they're already on it. Feel free to pull this bump in anyways and we can change this back when they fix it.

Comment (kanadgupta) [reply src/cmds/validate.ts:2]: eh I'll just convert this to a draft in the meantime, I'd rather keep the diff minimal

**Comment by kanadgupta** [comment]
see #739

**Pull request closed without merging**